docs/installation_guide.md

```diff
@@ -28,166 +30,357 @@ In addition, the deployment instructions on Kubernetes has been created by the c
-    - **host**: hostname for notary signer database
-    - **port**: port of notary signer database
-    - **db_name**: database name of notary signer database
-    - **username**: username to connect notary signer database
-    - **password**: password to notary signer database
-    - **ssl_mode**: is enable ssl mode
-  - **notary_server**:
-    - **host**: hostname for notary server database
-    - **port**: port of notary server database
-    - **db_name**: database name of notary server database
-    - **username**: username to connect notary server database
-    - **password**: password to notary server database
-    - **ssl_mode**: is enable ssl mode
-
-- **external_redis**: configs for use the external redis
-  - **host**: host for external redis
-  - **port**: port for external redis
-  - **password**: password to connect external host
-  - **registry_db_index**: db index for registry use
-  - **jobservice_db_index**: db index for jobservice
-  - **chartmuseum_db_index**: db index for chartmuseum
+|443|HTTPS|Harbor portal and core API accept HTTPS requests on this port. You can change this port in the configuration file.|
+|4443|HTTPS|Connections to the Docker Content Trust service for Harbor. Only required if Notary is enabled. You can change this port in the configuration file.|
+|80|HTTP|Harbor portal and core API accept HTTP requests on this port. You can change this port in the configuration file.|
+
+## Installation Procedure
+
+The installation procedure involves the following steps:
+
+1. Download the installer.
+2. Configure the **harbor.yml** file.
+3. Run the **install.sh** script with the appropriate options to install and start Harbor.
+
+## Download the Installer
+
+1. Go to the [Harbor releases page](https://github.com/goharbor/harbor/releases). 
+1. Select either the online or offline installer for the version you want to install.
+1. Use `tar` to extract the installer package:
+
+   - Online installer:<pre>bash $ tar xvf harbor-online-installer-<em>version</em>.tgz</pre>
+   - Offline installer:<pre>bash $ tar xvf harbor-offline-installer-<em>version</em>.tgz</pre>
+
+## Configure Harbor
+
+You set system level parameters for Harbor in the `harbor.yml` file that is contained in the installer package. These parameters take effect when you run the `install.sh` script to install or reconfigure Harbor. 
+
+After the initial deployment and after you have started Harbor, you perform additional configuration in the Harbor Web Portal. 
+
+### Required Parameters
+
+The table below lists the parameters that must be set when you deploy Harbor. At the very least, you must update the `hostname` parameter.
+
+**IMPORTANT**: Harbor does not ship with any certificates, and by default uses HTTP to serve registry requests. This is acceptable only in air-gapped test or development environments. In production environments, always use HTTPS. If you enable Content Trust with Notary, you must use HTTPS. 
```

Comment:
maybe change to 'in production environments where use should use Content Trust with Notary to properly sign all images, you must use HTTPS.'